moto/route53/responses.py

```diff
@@ -74,9 +75,11 @@ def sort_key(zone):
 
             zones = route53_backend.get_all_hosted_zones()
             zones = sorted(zones, key=sort_key)
+            template = Template(LIST_HOSTED_ZONES_BY_NAME_RESPONSE)
+            return 200, headers, template.render(zones=zones, dnsname=dnsname)
 
-        template = Template(LIST_HOSTED_ZONES_BY_NAME_RESPONSE)
-        return 200, headers, template.render(zones=zones)
+        # template = Template(LIST_HOSTED_ZONES_BY_NAME_RESPONSE)
```

Comment:
Looks like we can keep the return here, as the code is the same in the `if` and `else` sections

Reply:
Good eye, forgot to get rid of those. Done in 2abf91cab05ef7a903fb393018dd8bd0c6c656d4